webpack.common.config.js

```diff
@@ -23,9 +24,13 @@ module.exports = function (outputFilename) {
         'okta/underscore': SHARED_JS + '/util/underscore-wrapper',
         'okta/handlebars': 'handlebars/dist/handlebars',
         'okta/moment': 'moment/moment',
+        'okta/moment-tz': VENDOR + '/lib/moment-timezone-with-data-2010-2020',
         'okta/jqueryui': EMPTY,
         'okta': 'shared/util/Okta',
         'shared/util/Bundles': 'util/Bundles',
+        'mixpanel-browser': SHARED_JS + '/util/mixpanel-factory',
```

Comment:
We don't need mixpanel for RAAS. In the sign in widget mixpanel was listed in the EMPTY section as we are not using it. I had to add this because an internal courage component was using it and  would thrown an error when it tried to find it.

Reply:
Which internal component depends on mixpanel? Is it something that's used by the widget?

Reply:
yeah, previously courage required mixpanel, and we just pointed it to EMPTY. Can we do the same here? I'm curious to about which component needs it

Reply:
packages/@okta/courage/src/util/Metrics.js in courage is using mixpanel which came with the upgrade. Don't think its used by the widget. Let me try removing mixpanel-browser, selectize, vendor/plugins/jquery.autosuggest-1.4.min to see if I still see the error.

Reply:
Took me a while, but I got this error
 ERROR in ./target/js/shared/util/Metrics.js
Module not found: Error: Cannot resolve module 'mixpanel-browser' in /home/vagrant/okta/my-signin-widget/okta-signin-widget/target/js/shared/util
 @ ./target/js/shared/util/Metrics.js 4:0-246:2

Steps: 
1: deleted node_modules
2: npm install
3: npm start

Reply:
Okay - we probably shouldn't be using Metrics in the widget, can you just point that to empty and see if that resolves this?

Reply:
Btw, in general, if new components/etc are added to courage but they're not actively being used in the widget, we should add them to the blacklist here.

Reply:
I had to add `mixpanel-browser` to the EMPTY list instead of `mixpanel`. That seems to get rid of the error. Also moved selectize to EMPTY.

Reply:
I'd still say that we should axe out dependencies as high up the chain as possible. If we mark metrics as empty, we don't need to mark mixpanel-browser as empty (or any other independent dependencies that it would pull in).

Reply:
Looks like Metrics is already marked as empty https://github.com/okta/okta-signin-widget/pull/302/files/e8f77e3cf3c6128cd1f976b62e2abbbecdb7fd0f..4d0063d56cb70dd3a374e9434dc500fec0b0cb08#diff-aa4050a929bf9b482ff53091a6327b9fR63

Reply:
Tried moving Metrics and metrics-factory to EMPTY, but it still throws the error. Setting mixpanel-browser to EMPTY fixes the error.

Reply:
Why is this the case? I.e. if we are not loading metrics, it shouldn't try to load any metrics dependencies. So, is it:

1. Metrics is not being ignored even if we are swapping it out with empty 
2. Another file is requiring metrics

Reply:
@rchild-okta this is now addressed with the latest commit. I had to move mixpanel to the null loaders section